src/containers/permissionssummary/PermissionsSummaryReducer.js

```diff
@@ -91,6 +94,26 @@ export default function reducer(state :Immutable.Map<*, *> = INITIAL_STATE, acti
     case actionTypes.RESET_PERMISSIONS:
       return INITIAL_STATE;
 
+    case actionTypes.SET_ORGS_MEMBERS:
+      const orgMembersMerge = {
+        orgsMembers: {
+          [action.members.orgId]: action.members.members
+        }
+      }
+      return state.mergeDeep(orgMembersMerge);
```

Comment:
Avoid using `mergeDeep()`. We've had problems with it in the past.

Reply:
I know you've mentioned having issues before... what's the alternative that works in this case?
 merge(), I believe, will not properly merge details deeper than the parent state.